code/datums/spells/bloodcrawl.dm

```diff
@@ -32,3 +32,213 @@
 		return
 	revert_cast()
 	to_chat(user, "<span class='warning'>You are unable to blood crawl!</span>")
+
+
+//Travel through pools of blood. Slaughter Demon powers for everyone!
+#define BLOODCRAWL     1
+#define BLOODCRAWL_EAT 2
+
+/mob/living/proc/phaseout(var/obj/effect/decal/cleanable/B)
```

Comment:
I created a if(clown) check, where if you have the weapon you are having different version in some places. 
Specifically, if you are clown it does not check for empty hands or give bloodcrawl claws, it gives a different animation, and it has an interruptible delay for entering the spell. 
I removed the flick lines on the animation that made it overlap, as well as adding a small sleep on reappearing so the animation fully plays before you reappear.

Does that answer your question?